src/core/contract_ffi.ml

```diff
@@ -0,0 +1,55 @@
+(* type _ syscall =
```

Comment:
Why this is commented?

Reply:
This is just ffi to not lose it anywhere